serverless/messaging/how-to/create-manage-queues.mdx

```diff
@@ -1,115 +1,106 @@
-    <Lightbox src="scaleway-create-queue.webp" alt="" />
-
-4. Click the green `+` button to create a new queue. The queue creation wizard displays.
-    <Lightbox src="scaleway-create-queue-wizard.webp" alt="" />
-
-5. Choose your **queue type**, from one of the following:
+3. Choose your **queue type**, from one of the following:
     - **Standard**: Standard queues ensure at-least-once delivery. The exact order of messages in the queue is not preserved.
     - **FIFO**: FIFO queues (**F**irst **In** **F**irst **O**ut) preserves the exact ordering of messages.
 
-6. Enter a **name** for your queue. The name must be no more than 80 characters, and consist only of alphanumeric characters, hyphens, and underscores. Alternatively, you can go with the auto-generated name suggested for you.
+4. Enter a **name** for your queue. The name must be no more than 80 characters, and consist only of alphanumeric characters, hyphens, and underscores. Alternatively, you can go with the auto-generated name suggested for you.
 
-7. Configure your queue's **parameters**. The following parameters can be defined:
+5. Configure your queue's **parameters**. The following parameters can be defined:
     - **Visibility timeout**: The length of time (in seconds) during which, after a message is received, it is hidden and cannot be received again by the same or other consumers. This is useful as the queue itself does not automatically delete messages once they are received, and so prevents consumers from receiving the same message many times before they have finished processing it. Therefore, the value set here should be informed by the expected length of time it will take consumers to process messages. The default value is 30 seconds. The value must be between 1 second and 43 200 seconds (12 hours).
     - **Message retention period**: The length of time (in seconds) that messages are kept in the queue before being deleted. Setting a longer message retention period allows for a longer interval between a message being sent and it being received. The default value is 60 seconds. The value must be between 60 seconds and 1 290 600 seconds (14 days).
     - **Maximum message size**: The maximum size (in kilobytes) of a message that can be sent to the queue. Messages greater than the defined size will be rejected. The default value is 256 KB. The value must be between 1 KB and 256 KB.
     - **Content Based Deduplication**: This setting is only available for FIFO queues. Enable content based deduplication if the message body is guaranteed to be unique for each message. A unique hash value is generated from the body of each message, which is used as its deduplication ID. This avoids the need to set a deduplication ID when sending messages.
 
-8. Click **Create queue** to finish.
+7. Click **Create queue** to finish.
+
+You are returned to the **Queues** tab, where your newly-created queue now displays. You can view its URL here, for use in your API/CLI operations. 
```

Comment:
"Here" as in the "Queues" tab or should there be a link to where they will see the URL?

Reply:
I've clarified it in the text